sycl/test/basic_tests/stream.cpp

```diff
@@ -1,10 +1,9 @@
 // RUN: %clangxx -fsycl %s -o %t.out -lOpenCL
 // RUN: env SYCL_DEVICE_TYPE=HOST %t.out | FileCheck %s
 // RUN: %CPU_RUN_PLACEHOLDER %t.out %CPU_CHECK_PLACEHOLDER
-// RUN: %GPU_RUN_PLACEHOLDER %t.out %GPU_CHECK_PLACEHOLDER
+// RUN: %GPU_RUN_ON_LINUX_PLACEHOLDER %t.out %GPU_CHECK_ON_LINUX_PLACEHOLDER
```

Comment:
I think it will simpler to separate GPU run lines into another file.

Reply:
> I think it will simpler to separate GPU run lines into another file.

Alexey, I don't understand this comment. Why we would have 2 identical LIT tests?
If I and Artur understood you wrong, please give more detailed comment.

I like the current approach proposed by Artur.